net.solarnetwork.node.setup.web/web/WEB-INF/jsp/a/home.jsp

```diff
@@ -1,5 +1,17 @@
 <%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<style>
```

Comment:
All CSS should be moved to `/web/css/solarnode.css`. All these classes should be constrained to a parent CSS class namespace to group them together and avoid collisions with other areas of the webapp, like here where `.chart` is used as the parent namespace:

```css
.chart .x.axis line {
    shape-rendering: auto;
}

.chart .line {
    fill: none;
}
```